pluginconfig/pluginconfig-311.yaml

```diff
@@ -302,6 +302,48 @@ versions:
       sync: osarpint.azurecr.io/openshift-on-azure/azure:v12.0
       tlsProxy: osarpint.azurecr.io/openshift-on-azure/azure:v12.0
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10182019
+  v12.1:
```

Comment:
@kwoodson you need to set `pluginVersion: v12.1` at the top of this file, otherwise lgtm

Reply:
Updated.  Thanks